flexget/plugins/operate/cfscraper.py

```diff
@@ -27,8 +29,22 @@ def on_task_start(self, task, config):
         except ImportError as e:
             log.debug('Error importing cfscrape: %s' % e)
             raise plugin.DependencyError('cfscraper', 'cfscrape', 'cfscrape module required. ImportError: %s' % e)
+
+        class CFScrapeWrapper(cfscrape.CloudflareScraper, Session):
+            """
+            Wrapper class to strip unwanted input args from the Flexget requests class
+            """
+
+            def request(self, method, url, *args, **kwargs):
+                raise_status = kwargs.pop('raise_status', False)
```

Comment:
So, if we change our requests util to call `super` instead of `Session.request` directly, and reverse the order of the base classes here, can we just skip this stripping of arg, and our normal request method will strip it and handle our special stuff before passing to cloudflare session?